scripts/exregional_get_extrn_mdl_files.sh

```diff
@@ -89,7 +89,7 @@ yyyymmdd=${yyyymmddhh:0:8}
 mm=${yyyymmddhh:4:2}
 dd=${yyyymmddhh:6:2}
 hh=${yyyymmddhh:8:2}
-
+cyc=${hh}
```

Comment:
This is potentially a problem. It resets an NCO required variable. This is the GFS cyc that we're looking for, which is a namespace collision  with the existing `cyc` for RRFS.

Reply:
removed.